### DISCUSSION TITLE
Non-admin users cannot view knowledge collection source documents

### DISCUSSION THREAD

**Discussion by InventoCasa** [open-webui/open-webui#7578] [Ideas]
## Installation Method

Docker

## Environment

- **Open WebUI Version:** v0.4.7

- **Operating System:** Ubuntu 22.04
- **Browser (if applicable):** Firefox, Chrome

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

When prompting with a Knowledge Connection and clicking on "Citation" --> "Source" in the answer footer, only the admin user is able to View the document. Other users will get a 404 not found error according to docker container logs.

## Actual Behavior:

Non-admin users should be able to view the source document, if they have access to the specific knowledge collection the source document is part of.

## Description

**Bug Summary:**
Non-admin users cannot view knowledge collection source documents

## Reproduction Details

**Steps to Reproduce:**
See "actual behaviour"

**Comment by InventoCasa** [comment]
@tjbck why was this moved to a discussion?

**Reply by InventoCasa** [reply]
@tjbck I don't get why this was converted to a discussion. It's a bug, especially in combination with the new groups feature.

**Reply by ivanbaldo** [reply]
Here is an issue https://github.com/open-webui/open-webui/issues/11331 and the PR https://github.com/open-webui/open-webui/pull/11087 .

**Comment by ivanbaldo** [comment]
It doesn't make sense that a user is able to chat with a knowledge base, see the matching chunks text, but it's not able to consult the source document...
